codersdk/provisionerdaemons.go

```diff
@@ -39,17 +39,40 @@ const (
 	LogLevelError LogLevel = "error"
 )
 
+// ProvisionerDaemonStatus represents the status of a provisioner daemon.
+type ProvisionerDaemonStatus string
+
+// ProvisionerDaemonStatus enums.
+const (
+	ProvisionerDaemonOffline ProvisionerDaemonStatus = "offline"
+	ProvisionerDaemonIdle    ProvisionerDaemonStatus = "idle"
+	ProvisionerDaemonBusy    ProvisionerDaemonStatus = "busy"
+)
+
 type ProvisionerDaemon struct {
-	ID             uuid.UUID         `json:"id" format:"uuid"`
-	OrganizationID uuid.UUID         `json:"organization_id" format:"uuid"`
-	KeyID          uuid.UUID         `json:"key_id" format:"uuid"`
-	CreatedAt      time.Time         `json:"created_at" format:"date-time"`
-	LastSeenAt     NullTime          `json:"last_seen_at,omitempty" format:"date-time"`
-	Name           string            `json:"name"`
-	Version        string            `json:"version"`
-	APIVersion     string            `json:"api_version"`
-	Provisioners   []ProvisionerType `json:"provisioners"`
-	Tags           map[string]string `json:"tags"`
+	ID             uuid.UUID         `json:"id" format:"uuid" table:"id"`
+	OrganizationID uuid.UUID         `json:"organization_id" format:"uuid" table:"organization id"`
+	KeyID          uuid.UUID         `json:"key_id" format:"uuid" table:"-"`
+	CreatedAt      time.Time         `json:"created_at" format:"date-time" table:"created at"`
+	LastSeenAt     NullTime          `json:"last_seen_at,omitempty" format:"date-time" table:"last seen at"`
+	Name           string            `json:"name" table:"name,default_sort"`
+	Version        string            `json:"version" table:"version"`
+	APIVersion     string            `json:"api_version" table:"api version"`
+	Provisioners   []ProvisionerType `json:"provisioners" table:"-"`
+	Tags           map[string]string `json:"tags" table:"tags"`
```

Comment:
I do not love this output: `map[owner: scope:organization]`, but I'm ok with it.

We could have the table formatter know how to handle some basic types like `map[string]string`. Just leave it for this PR though.

Reply:
I had a similar thought, something human-readable and copy/paste friendly:

```
owner = (none)
scope = organization
```